en/docs/setup/configuring-a-sp-and-idp-using-configuration-files.md

```diff
@@ -2,100 +2,101 @@
+![Configuring a SP and IdP Using Configuration Files](../assets/img/103329463/103329465.png)
 
 The following are the high level steps required for this scenario.
 
-1.  Add the **`           identity provider IS          `** in the
+1.  Add the **`           identity provider IS          `** to the
     **`           service provider IS          `** as an identity
     provider.
-2.  Add the **`           service provider IS          `** in the
+2.  Add the **`           service provider IS          `** to the
     **`           identity provider IS          `** as a service
     provider.
-3.  Add the travelocity.com application in the
+3.  Add the `travelocity.com` application in the
     **`           service provider IS          `** as a service
     provider. This uses the identity provider created earlier (in
     step 1) as a federated authenticator.
 
-The above processes can be easily done using the [Management
-Console](../../setup/getting-started-with-the-management-console), but the
-service provider and identity provider created in
-**`          service provider IS         `** are only visible to the
-tenant who creates them. So the difference here is that the identity
-provider and service provider in service provider IS are created using
-configuration files so that they are available to all the tenants
-in service provider IS.
+!!! note
+    The above processes can be easily done using the [Management
+    Console](../../setup/getting-started-with-the-management-console), but the
+    `service provider` and `identity provider` created in
+    **`          service provider IS         `**, will **only be visible to the
+    tenant who creates them**. 
+    
+    Therefore, difference here is that the `identity
+    provider` and `service provider` in **`service provider IS`** are created using
+    configuration files, is **available to all the tenants**
+    in **`service provider IS`**.
 
 The following sections provide instructions on how to carry out the
 above steps.
 
--   [Adding the service provider in the identity provider
-    IS](#ConfiguringaSPandIdPUsingConfigurationFiles-AddingtheserviceproviderintheidentityproviderIS)
--   [Adding the identity provider in the service provider
-    IS](#ConfiguringaSPandIdPUsingConfigurationFiles-AddingtheidentityproviderintheserviceproviderIS)
--   [Adding the service provider in the service provider
-    IS](#ConfiguringaSPandIdPUsingConfigurationFiles-AddingtheserviceproviderintheserviceproviderIS)
--   [Running the travelocity
-    application](#ConfiguringaSPandIdPUsingConfigurationFiles-Runningthetravelocityapplication)
--   [Testing with
-    tenants](#ConfiguringaSPandIdPUsingConfigurationFiles-Testingwithtenants)
+-   Adding the service provider in the identity provider IS
+-   Adding the identity provider in the service provider IS
+-   Adding the service provider in the service provider IS
+-   Running the travelocity application
+-   Testing with tenants
```

Comment:
You can remove this TOC as it will be available through right-hand navigation.

Reply:
fixed: 36c85c6dbb7747a3519150db3e2f3302d55bbc54